app/scripts/metamask-controller.js

```diff
@@ -2501,13 +2511,12 @@ export default class MetamaskController extends EventEmitter {
     if (!keyring.mnemonic) {
       throw new Error('Primary keyring mnemonic unavailable.');
     }
-    return keyring.mnemonic;
+    return uint8ArrayMnemonicToString(keyring.mnemonic);
```

Comment:
`getPrimaryKeyringMnemonic` is used by Snaps which still passes mnemonics around as strings. They will be refactoring to pass mnemonics as Uint8Arrays but until then we must recover and return the string version from the Uint8Array format that is now on the `keyring` value returned by the `KeyringController` on line 2508 as of this latest version bump.